bsdfdisk/edit_disk.cgi

```diff
@@ -23,9 +23,9 @@ push(@info, &text('disk_dsize', &nice_size($disk->{'size'})));
 if ($disk->{'model'}) {
         push(@info, &text('disk_model', $disk->{'model'}));
         }
-push(@info, &text('disk_cylinders', $disk->{'cylinders'}));
```

Comment:
@karmantyu, why deleting this line? Please remember we need to provide support for older versions!